generators/server/templates/dotnetcore/src/Project/appsettings.json.ejs

```diff
@@ -90,7 +90,8 @@
                       //}
                 }    
                 <%_ } _%>
-            }
+            },
+            "EnforceHttps":  false
```

Comment:
Mayve you can set EnforceHttps to false in https://github.com/jhipster/jhipster-dotnetcore/blob/main/generators/server/templates/dotnetcore/src/Project/appsettings.Development.json.ejs
and set to true in https://github.com/jhipster/jhipster-dotnetcore/blob/main/generators/server/templates/dotnetcore/src/Project/appsettings.Production.json.ejs